contracts/contracts/Sapphire.sol

```diff
@@ -14,12 +14,17 @@ library Sapphire {
         0x0100000000000000000000000000000000000003;
     address private constant DECRYPT =
         0x0100000000000000000000000000000000000004;
-    address private constant GENERATE_KEYPAIR =
+    address private constant GENERAGE_SIGNING_KEYPAIR =
```

Comment:
Should be `GENERATE_SIGNING_KEYPAIR`

Reply:
Indeed! Can you open up a PR with the fix? Thanks a lot!